interface/src/app/gene/network/network.js

```diff
@@ -198,7 +198,8 @@ angular.module('adage.gene.network', [
               edgeTip.show(data, target);
             },
             function error(err) {
-              $log.error('Failed to get node info for gene edge: ' + err);
+              $log.error(
+                'Failed to get node info for gene edge: ' + err.statusText);
```

Comment:
Nice! I agree that we should use a '.' in our data binding on the template, especially given what we have recently learned about scopes.